index.bs

```diff
@@ -532,6 +532,13 @@ that agent.  Agents should discard any authentication message whose
 `auth-initation-token` is set and does not match the `at` provided by the
 advertising agent.
 
+In the `psk-min-bits-of-entropy` field of the [=auth-capabilities=] messsage,
+agents may specify the minimum bits of entropy it requires for a PSK, in the
+range of 20 to 60 bits inclusive.  Agents must terminate any connection
+attempting to authenticate with a PSK with fewer than the minimum number of bits
+of entropy and must not attempt to authenticate with an agent that advertises a
+value outside the allowed range.
```

Comment:
The text doesn't spell out how the agents agree on the bit length of the PSK.  I'm assuming that the PSK presenter is required to generate a PSK of length max([my psk-min-bits-of-entry], [your psk-min-bits-of-entropy]).   I think the text should include this rule.

Reply:
OK, I added that.

Reply:
FYI, I also added a default value.